war/src/main/java/io/lumeer/engine/rest/CollectionService.java

```diff
@@ -121,38 +118,71 @@ public void init() {
     */
    @GET
    @Path("/")
-   public List<String> getAllCollections() {
-      List<String> collections = new ArrayList<>();
+   public List<Collection> getCollections(@QueryParam("collection") String collectionNamePart, @QueryParam("limit") int limit) {
```

Comment:
Why do we need to query collections by partial name? `SearchService` is responsible for such queries or am I missing something?